educational-webapp/src/main/webapp/script.js

```diff
@@ -1,3 +1,81 @@
-function onClick(){
-    console.log('Letter');
+var aAudio = new Audio('sounds/A.wav');
+var bAudio = new Audio('sounds/B.wav');
+var cAudio = new Audio('sounds/C.wav');
+var dAudio = new Audio('sounds/D.wav');
+var eAudio = new Audio('sounds/E.wav');
+var fAudio = new Audio('sounds/F.wav');
+var gAudio = new Audio('sounds/G.wav');
+var hAudio = new Audio('sounds/H.wav');
+var iAudio = new Audio('sounds/I.wav');
+var jAudio = new Audio('sounds/J.wav');
+var kAudio = new Audio('sounds/K.wav');
+var lAudio = new Audio('sounds/L.wav');
+var mAudio = new Audio('sounds/M.wav');
+var nAudio = new Audio('sounds/N.wav');
+var oAudio = new Audio('sounds/O.wav');
+var pAudio = new Audio('sounds/P.wav');
+var qAudio = new Audio('sounds/Q.wav');
+var rAudio = new Audio('sounds/R.wav');
+var sAudio = new Audio('sounds/S.wav');
+var tAudio = new Audio('sounds/T.wav');
+var uAudio = new Audio('sounds/U.wav');
+var vAudio = new Audio('sounds/V.wav');
+var xAudio = new Audio('sounds/X.wav');
+var yAudio = new Audio('sounds/Y.wav');
+var zAudio = new Audio('sounds/Z.wav');
+
+function alphabetAudio(letter) {
+    if(letter == 'a') {
+        aAudio.play();
+    } else if(letter == 'b') {
```

Comment:
You can get a bit clever here if instead of keeping aAudio, bAudio, etc, you keep an array of size 26 where index 0 represents "a" and index 1 represents "b", etc.

```
sounds = [new Audio('sounds/A.wav'), new Audio('sounds/b.wav'), new Audio('souds/c.wav', ...]
```

You're also able to convert any single character to its [ASCII integer value](http://www.asciitable.com/). 

The lowercase alphabet starts at 97 ('a' is 97, 'b' is 98, and so on) so here you can do this:
```
function alphabetAudio(letter) {
    alphaIndex = letter.toLowerCase().charCodeAt(0) - 97;
    sounds[alphaIndex].play();
}
```

`toLowerCase()` converts your letter to lower case, just in case it comes in as upper case
`charCodeAt()` returns the ASCII value for your character
`- 97` is to make 'a' be index 0, because the lowercase alphabet starts at ASCII value 97

Reply:
I am going to merge now so Aileen can have access to the sound files, and will work on this in a separate PR. Thanks for this advice!